tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -167,6 +178,53 @@
 # It is quite clear that the MRI surfaces (head, brain) are not well aligned
 # to the head digitization points (dots).
 #
+# Compute the transform
+# ---------------------
+# Let's step through the process that the transform is accomplishing.
+
+
+def plot_alignment(points):
```

Comment:
Please call it `plot_alignment_custom` or something to avoid name confusion with `mne.viz.plot_alignment`